h/templates/deform/checkbox_choice.jinja2

```diff
@@ -14,8 +14,11 @@
              {%- if field.error -%}
              aria-invalid="true"
              {% endif -%}
+             {%- if field.widget.disable_autocomplete -%}
+             autocomplete="off"
+             {% endif -%}
```

Comment:
I wonder if we should enable this option globally to prevent the same issue from occurring in other contexts?

Reply:
For other fields like password or username, you want to have the autocomplete on (https://github.com/hypothesis/h/pull/6654).

I think this mostly affects checkboxes.

Reply:
I should clarify - I was thinking about checkboxes specifically.

Reply:
I have made `autocomplete="off"` default.